CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.14.0)
+cmake_minimum_required(VERSION 3.17.2)
```

Comment:
The intent is to match the vulkan validation layers and reduce friction for developers who work on multiple CMake repos.

Reply:
Also the vulkan loader, vulkan extension layer, etc. also use 3.17.2 as the minimum